templates/_includes/toc.html

```diff
@@ -1,4 +1,4 @@
 - Contents
 {:toc}
 
-{::options toc_levels="3,4" /}
```

Comment:
@araiford, please don't let me forget that I need to fix the TOC not appearing in Helix rows issue when I set up the heading styles fix...

Reply:
will do